paddlecloud/paddlejob/views.py

```diff
@@ -42,7 +44,9 @@ def file_publish_view(request):
     logging.info("downloading file from: %s, record(%s)", real_path, record.path)
 
     # mimetype is replaced by content_type for django 1.7
-    response = HttpResponse(open(real_path), content_type='application/force-download') 
+    chunk_size = 8192
+    response = StreamingHttpResponse(FileWrapper(open(real_path, 'rb'), chunk_size),
+                           content_type=mimetypes.guess_type(real_path)[0])
     response['Content-Disposition'] = 'attachment; filename=%s' % os.path.basename(record.path)
```

Comment:
Can you please also add `Content-Length` please?

Reply:
Following the doc: https://docs.djangoproject.com/en/1.8/ref/request-response/#django.http.StreamingHttpResponse
> StreamingHttpResponse should only be used in situations where it is absolutely required that the whole content isn’t iterated before transferring the data to the client. Because the content can’t be accessed, many middlewares can’t function normally. For example the ETag and Content- Length headers can’t be generated for streaming responses.

We don't need the Content-Length header attribution.